node/src/components/storage.rs

```diff
@@ -704,7 +706,8 @@ impl Storage {
                 let hash = block.hash();
                 let finality_signatures = match self.get_finality_signatures(&mut txn, hash)? {
                     Some(signatures) => signatures,
-                    None => BlockSignatures::new(*hash, block.header().era_id()),
```

Comment:
@darthsiroftardis @marc-casperlabs Ditto here.  This should be merged upstream...